components/button/appearance.md

```diff
@@ -10,151 +10,111 @@ position: 35
 
 # Appearance Settings
 
-You can control the appearance of the button by setting the following attributes:
+The Telerik Button component for Blazor provides several ways to control its appearance. This article discusses the following component parameters:
 
 * [FillMode](#fillmode)
 * [Rounded](#rounded)
 * [Size](#size)
 * [ThemeColor](#themecolor)
 
-You can use all of them together to achieve the desired appearance. This article will explain their effect one by one.
+You can use all of them together to achieve the desired appearance.
 
 ## FillMode
 
-The `FillMode` toggles the background and border of the TelerikButton. You can set the parameter to a member of the `Telerik.Blazor.ThemeConstants.Button.FillMode` class:
+The `FillMode` parameter changes or removes the background and border of the Button. Set the parameter to a member of the static class `Telerik.Blazor.ThemeConstants.Button.FillMode`. When setting the `FillMode` parameter value inline, start with a `@` character, otherwise the predefined string constant cannot be resolved.
```

Comment:
```suggestion
The `FillMode` parameter changes or removes the background and border of the Button. Set the parameter to a member of the static class `Telerik.Blazor.ThemeConstants.Button.FillMode`. When setting the `FillMode` parameter value inline, start with an `@` character, otherwise the predefined string constant cannot be resolved.
```